Add ResearchForDevelopmentOutputs Finder

edwardkerry opened the pull request:
DFID is being merged into the new FCDO. This PR adds a new `ResearchForDevelopmentOutputs` Finder based on   `DFIDResearchOutputs`. 

[govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas/pull/1001)
[search-api](https://github.com/alphagov/search-api/pull/2171)

[Trello](https://trello.com/c/wye58SZu/2109-update-x2-dfid-specialist-finders-to-fcdo)

AlanGabbianelli commented on lib/documents/schemas/international_development_funds.json:
```diff
@@ -3,14 +3,15 @@
   "format_name": "International development funding",
   "name": "International development funding",
-  "description": "Find and apply for funds to run international development projects",
+  "description": "Grants published before XX September 2020 were published by the Department for International Development (DFID)
```
Will `XX September 2020` be changed to an actual date in a subsequent PR?

edwardkerry replied:
yes, there are to-do's on the trello card and this is not to merge until then

AlanGabbianelli replied:
Probably worth updating PR title to `[DO NOT MERGE]Updated DFID document types to FCDO`

Commit:
Add Research For Development Output finder

This new Finder is based on the DfidResearchOutput finder and will
be used to as replacement with a new document type

thomasleese commented on lib/documents/schemas/research_for_development_outputs.json:
```diff
@@ -0,0 +1,987 @@
+{
+  "content_id": "af7c7cdb-1335-4f1c-b21c-b3c8bfec8a1b",
```
I think this is the same as the old finder.

thomasleese commented on lib/documents/schemas/research_for_development_outputs.json:
```diff
@@ -0,0 +1,987 @@
+  "filter": {
+    "format": "research_for_development_output"
+  },
+  "signup_content_id": "a062704c-f49d-4942-aaf4-c06d81ada3b8",
```
Same here as the old finder, I think it'll need to be changed.

edwardkerry replied:
Does this need to relate to anything else, or just a new UUID?
